rwgame/RWGame.cpp

```diff
@@ -67,14 +68,20 @@ RWGame::RWGame(int argc, char* argv[])
 	;
 
 	po::variables_map vm;
-	po::store(po::parse_command_line(argc, argv, desc), vm);
-	po::notify(vm);
+	try
+	{
+		po::store(po::parse_command_line(argc, argv, desc), vm);
+		po::notify(vm);
+	}
+	catch (po::unknown_option& ex)
+	{
+		help = true;
+	}
 
-	if( vm.count("help") )
+	if( help || vm.count("help") )
 	{
-		// TODO: This is a hack
 		std::cout << desc << std::endl;
-		throw std::runtime_error("Terminate");
+		throw std::invalid_argument("");
```

Comment:
Yeah, as I tried to explain, the exception is just a crutch to exit the game from the RWGame constructor.

I've added the unrecognized option text. It will only show the first "wrong" option, but so does e.g. cat. For example:
```
$ cat -a -b
cat: invalid option -- 'a'
```

The only thing that's left is the return code. It will indeed be the same for both a wrong command line argument and `--help`. This is a bit unfortunate, but would be easier to fix after reorganizing the code a bit.